src/main/java/org/terasology/projsndwv/genetics/components/GeneticsComponent.java

```diff
@@ -20,23 +20,47 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/***
+ * Stores a genome where one allele at a locus is expressed and the other is repressed.
+ */
 public final class GeneticsComponent implements Component {
+    /**
+     * Number of loci in this genome.
+     */
     public int size;
 
+    /**
+     * List of expressed alleles.
+     */
     public List<Integer> activeGenes;
+    /**
+     * List of repressed alleles.
+     */
     public List<Integer> inactiveGenes;
 
+    /**
+     * Constructs a {@code GeneticsComponent} with no size. Intended only for loading.
```

Comment:
It may be worth briefly expanding on this - I am figuring by reading between the lines that "Intended only for loading" is referring to how we have to include a parameter-less constructor in a component for it to serialize right. That may be good to point out in a parenthesis or so "(to serialize correctly all components must have a parameter-less constructor)"